src/config/static-pages.php

```diff
@@ -0,0 +1,7 @@
+<?php
+return [
+    'url' => 'page-list',
```

Comment:
This route is not for admin but for frontend,
Also i suggest to name the key as `route` because it represents the route name but not url.